boot/zephyr/Kconfig

```diff
@@ -289,6 +289,26 @@ config BOOT_ENCRYPT_X25519
 	  described under "ECIES-X25519 encryption" in docs/encrypted_images.md.
 endif # !SINGLE_APPLICATION_SLOT
 
+config BOOT_ENCRYPT_KEY_FILE
```

Comment:
I believe BOOT_USE_ENCRYPT_KEY_FILE should be moved inside `#if !SINGLE_APPLICATION_SLOT`.

Reply:
You are correct. Moved inside `#if !SINGLE_APPLICATION_SLOT`